floris/tools/floris_interface.py

```diff
@@ -631,6 +639,10 @@ def get_farm_power(
         # for turbine in self.floris.farm.turbines:
         #     turbine.use_turbulence_correction = use_turbulence_correction
 
+        # Confirm calculate wake has been run
+        if self.floris.state is not State.USED:
```

Comment:
@paulf81 I think we do not need this here since it would be caught by `self.get_turbine_powers()`. Do you agree?

Reply:
that makes sense to me, maybe this could be deleted and then we can make a test we get an error if calc wake not run before get_turbine_powers?